src/main/java/com/ibm/infrastructure/scanning/repositories/Scan.java

```diff
@@ -60,6 +61,8 @@ class Scan extends PanacheEntityBase {
     @Nonnull public String revision;
     @Nullable public String commitHash;
     @Nullable public String subFolder;
+    @Nullable public String purl;
+    @Nullable public String tag;
```

Comment:
Do we still need to explicitly store the tag here?

Reply:
We don't. I store branch/tag in `revision`.